src/diffusers/loaders.py

```diff
@@ -752,6 +753,18 @@ def enable_lora(self):
             raise ValueError("PEFT backend is required for this method.")
         set_adapter_layers(self, enabled=True)
 
+    def delete_lora(self, adapter_name):
```

Comment:
also what about `delete_adapters` ? To make it in-line with `set_adapters` wdyt @BenjaminBossan @pacman100

Reply:
Good question. AFAICT, there is a bit of a mixture of using the term "lora" or "adapter". I think it depends on whether there is the plan to later enhance this method to support other adapter types. If this is renamed, then `delete_lora_layers` should also be renamed.

When it comes to whether to allow passing a list of names: I see no harm in it.

Reply:
I think `delete_adapters()` is nice. In-line with `set_adapters()` too. 

Could you also include the type-annotation here?

Reply:
Sure ok I'll go with that!

Reply:
Could it make sense to allow for deleting multiple LoRAs one-shot like we provide the utility (`set_adapters()`) to handle single adapter loading and multi-adapter loading?  This promotes better feature parity no?

Reply:
But we have enable_lora, disable_lora, unload_lora_weights, load_lora_weights